src/Domain/CMakeLists.txt

```diff
@@ -39,5 +39,6 @@ target_link_libraries(
   INTERFACE CoordinateMaps
   INTERFACE DataStructures
   INTERFACE ErrorHandling
+  INTERFACE LinearOperators
```

Comment:
Just out of curiosity, what in domain needs linear operators?

Reply:
The `size_of_element` function I added in PR #999 uses `mean_value_on_boundary`.